src/utils/update/managing.officer.mapper.ts

```diff
@@ -26,7 +26,7 @@ export const mapToManagingOfficer = (officer: CompanyOfficer): ManagingOfficerIn
     service_address: service_address,
     start_date: mapInputDate(officer.appointedOn),
     occupation: officer.occupation,
-    role_and_responsibilities: officer.officerRole,
+    role_and_responsibilities: "",
```

Comment:
I reckon we want undefined if we're not mapping a field's value

Reply:
Going with empty string after discussion on teams and UI validation testing checked.